cmd/ovirt-populator/go.mod

```diff
@@ -0,0 +1,73 @@
+module forklift.konveyor.io/ovirt-populator
+
```

Comment:
the build stuff should move to the `hack` folder, how about `hack/ovirt-populator`?